### PR TITLE
Integrate toolkit

### PR DESCRIPTION
**Pull request by rhernand3z**
- Added universal loader in parent container
- Added inert library -- @ricardoom recommended for locking DOM on a loading state
- Added toolkit and updated components to use toolkit grid and styles
- Added styles for header, sidebar nav and main forms

### CODE REVIEW

**Comment by rhernand3z** [comment]
@ricardoom -- see -- http://nypl-recap-admin-dev.us-east-1.elasticbeanstalk.com/update-metadata

**Comment by ktp242** [comment]
Is UI error handling in the scope of this PR?

File: src/shared/containers/TransferMetadata/TransferMetadata.jsx [hunk 1]
```diff
@@ -102,6 +111,7 @@ class TransferMetadata extends Component {
   * @param {object} event - contains the current event context of the input field
   */
   handleInputChange(event) {
+    console.log(event.target);
```
Comment (ktp242) [inline src/shared/containers/TransferMetadata/TransferMetadata.jsx]: Do we need the console.log here?

File: src/shared/components/FormField/FormField.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
```
Comment (ktp242) [inline src/shared/components/FormField/FormField.jsx:3]: Should we use `_isEmpty` as alias here like we used with underscore?

**Comment by ktp242** [comment]
Looks good! I just am not sure if I need to test error handling in this PR and how much I should test. I have some minor questions only. Up to you and Stephen to decide what to implement :-)

**Comment by ricardoom** [comment]
need some help running locally...

**Review by ktp242 — APPROVED** [review]

**Review by ricardoom — APPROVED** [review]
From our convo yesterday, I approve this PR

**Comment by rhernand3z** [comment]
You should now be able to login on AWS via your SSO user/pass. Closing and merging.

**Pull request merged**